.docz/gatsby-config.js

```diff
@@ -0,0 +1,112 @@
+const { mergeWith } = require('lodash/fp')
+const fs = require('fs-extra')
+
+let custom = {}
+const hasGatsbyConfig = fs.existsSync('./gatsby-config.custom.js')
+
+if (hasGatsbyConfig) {
+  try {
+    custom = require('./gatsby-config.custom')
+  } catch (err) {
+    console.error(
+      `Failed to load your gatsby-config.js file : `,
+      JSON.stringify(err),
+    )
+  }
+}
+
+const config = {
+  pathPrefix: '/',
+
+  siteMetadata: {
+    title: 'React Table Documentation',
+    description: 'Documentation for the react-table library built in Docz',
+  },
+  plugins: [
+    {
+      resolve: 'gatsby-theme-docz',
+      options: {
+        themeConfig: {},
+        themesDir: 'src',
+        mdxExtensions: ['.md', '.mdx'],
+        docgenConfig: {},
+        menu: [
+          'Getting Started',
+          'Installation',
+          'Concepts',
+          'Quick Start',
+          {
+            name: 'Examples',
+            menu: ['Simple', 'Complex', 'Controlled', 'UI/Rendering'],
+          },
+          { name: 'API', menu: ['Overview', 'useTable'] },
+          'FAQ',
+          'Contributing',
+          'Code of Conduct',
+          'Typescript',
+          'Changelog',
+        ],
+        mdPlugins: [],
+        hastPlugins: [],
+        ignore: ['docs/README.md', 'examples/**/*.md'],
+        typescript: false,
+        ts: false,
+        propsParser: false,
+        'props-parser': true,
+        debug: false,
+        native: false,
+        openBrowser: false,
+        o: false,
+        open: false,
+        'open-browser': false,
+        root: '/Users/jasonclark/Source/react-table/.docz',
+        base: '/',
+        source: './',
+        src: './',
+        files: '**/*.{md,markdown,mdx}',
+        public: '/public',
+        dest: '.docz/dist',
+        d: '.docz/dist',
+        editBranch: 'master',
+        eb: 'master',
+        'edit-branch': 'master',
+        config: '',
+        title: 'React Table Documentation',
+        description: 'Documentation for the react-table library built in Docz',
+        host: 'localhost',
+        port: 3002,
+        p: 3000,
+        separator: '-',
+        paths: {
+          root: '/Users/jasonclark/Source/react-table',
```

Comment:
Why exactly are this vendored to your computer?

Reply:
That is an outstanding question. I think that file is autogenerated (as is everything in the .docz folder).

Reply:
Interesting! Well, let's hope it doesn't cause any issues :)